docs/lib-documentation.md

```diff
@@ -80,6 +80,19 @@ Repartitions the network so that the connections between services match the spec
 * `partitionConnections`: Definitions of the connection state between the new partitions. If a connection between two partitions isn't defined in this map, the default connection will be used. Connections are not directional, so an error will be thrown if the same connection is defined twice (e.g. `Map[A][B] = someConnectionInfo`, and `Map[B][A] = otherConnectionInfo`).
 * `defaultConnection`: The network state between two partitions that will be used if the connection isn't defined in the partition connections map.
 
+### waitForEndpointAvailability(ServiceID serviceId, uint32 port, String path, uint32 initialDelaySeconds, uint32 retries, uint32 retriesDelayMilliseconds, String bodyText)
+Checks if a service is available. It executes several HTTP calls to and specific endpoint that has to be set with the arguments
+
+**Args**
+
+* `serviceId`: The ID of the service to check.
+* `port`: The port of the service to check. For instance 8080
```

Comment:
Recommendation: you'll be writing a lot of "For instance X" throughout your Kurtosis work, and rather than breaking into a new sentence I'd recommend instead doing `(e.g. X)`. The benefit is that you can drop it in the middle of a sentence, like so: `You'll want to blah blah blah, and provide a port (e.g. 8080) so that you can  blah blah blha....`

Reply:
Also, this would more accurately be "the port of the endpoint to check" (rather than "service")